2008-06-06  Nathan Froyd  <froydnj@codesourcery.com>

       * config/rs6000/rs6000.c (rs6000_mode_dependent_address): Remove
       PRE_INC and PRE_DEC cases.

Index: gcc/config/rs6000/rs6000.c
===================================================================
--- gcc.orig/config/rs6000/rs6000.c	2008-09-10 17:07:35.000000000 -0300
+++ gcc/config/rs6000/rs6000.c	2008-09-10 17:07:50.000000000 -0300
@@ -4245,8 +4245,7 @@
     case LO_SUM:
       return true;
 
-    case PRE_INC:
-    case PRE_DEC:
+    /* Auto-increment cases are now treated generically in recog.c.  */
     case PRE_MODIFY:
       return TARGET_UPDATE;
 
